temba/flows/models.py

```diff
@@ -2223,6 +2217,48 @@ class Meta:
         ordering = ("-modified_on",)
 
 
+@receiver(post_save, sender=Flow)
+def update_related_flows(sender, instance, created, **kwargs):
+    dependent_flows = instance.dependent_flows.all()
+    if created:
+        return
+
+    def flow_dependency_filter(dependency):
+        if dependency.get("type") == "flow":
+            return dependency.get("uuid") == instance.uuid
+        return False
+
+    def flow_node_filter(node):
+        if node["actions"] and node["actions"][0]["type"] == "enter_flow":
```

Comment:
Here is missing "Start somebody else" flow step

Reply:
Got it. Thanks